src/main/java/com/basho/riak/client/core/query/timeseries/TableDefinition.java

```diff
@@ -0,0 +1,85 @@
+package com.basho.riak.client.core.query.timeseries;
+
+import java.util.List;
+
+/**
+ * Holds a definition for a Table in Time Series Riak.
+ * Immutable once created.
+ * @author Alex Moore <amoore at basho dot com>
+ * @since 2.0.4
+ */
+public class TableDefinition
```

Comment:
Might be useful to add  the following utilitarian accessors for getting:
 - List of Partition Keys in proper order
 - List of Local Keys in proper order
 - Description by Column Name